packages/block-library/src/rss/block.json

```
@@ -39,6 +39,14 @@
 		"excerptLength": {
 			"type": "number",
 			"default": 55
+		},
+		"openInNewTab": {
+			"type": "boolean",
+			"default": false
+		},
+		"linkRel": {
```

```suggestion
		"rel": {
```

I just remembered that the Social Icons block also has a custom rel handler; let's make it consistent.

https://github.com/WordPress/gutenberg/blob/bb9028dc46c8422c9319c7d04149ad21312e30d0/packages/block-library/src/social-link/block.json#L22-L24